includes/Database_Upgrader.php

```diff
@@ -103,6 +102,7 @@ public function __construct( Injector $injector ) {
 	 */
 	public function register(): void {
 		add_action( 'admin_init', [ $this, 'run_upgrades' ], 5 );
+		add_action( 'cli_init', [ $this, 'run_upgrades' ], 5 );
```

Comment:
Can you give some context here?

Reply:
Figured it might make sense to run upgrades also when accessed via CLI

Reply:
On every CLI command. This might add overhead to CLI that users might not expect. 

Intead of this, we should have CLI commands to do this.

Reply:
Right, good point, I'll remove again for now